### ISSUE TITLE
TypeError: element.datepicker is not a function

### ISSUE THREAD

**Issue by bearmingo** [angular-ui/ui-date#125]
TypeError: element.datepicker is not a function
    at Object.link.initDateWidget [as fn] (date.js:102)
    at Scope.$get.Scope.$digest (angular.js:12650)
    at Scope.$get.Scope.$apply (angular.js:12915)
    at done (angular.js:8450)
    at completeRequest (angular.js:8664)
    at XMLHttpRequest.xhr.onreadystatechange (angular.js:8603)

Sometimes js print this exception, when I use this library.

**Comment by VictoWork** [comment]
Error: element.datepicker is not a function
.link/initDateWidget@http://localhost:9000/bower_components/angular-ui-date/src/date.js:102:13
$RootScopeProvider/this.$get</Scope.prototype.$digest@http://localhost:9000/bower_components/angular/angular.js:15683:23
$RootScopeProvider/this.$get</Scope.prototype.$apply@http://localhost:9000/bower_components/angular/angular.js:15951:13
done@http://localhost:9000/bower_components/angular/angular.js:10364:36
completeRequest@http://localhost:9000/bower_components/angular/angular.js:10536:7
requestLoaded@http://localhost:9000/bower_components/angular/angular.js:10477:1
 undefined

**Comment by nwhittaker** [comment]
I experienced a similar error message when loading "angular.js" before "jquery.js". Flipping the load order fixed it. Possibly something similar going on here? Are you using AMD?

**Comment by tjwoon** [comment]
This directive depends on jQuery-UI plugin, so you have to include jQuery **before** Angular, so that Angular will use jQuery instead of the built in jqLite.

TL;DR; Include jquery.js **before** angular.js

**Comment by abatista667** [comment]
Thanks

**Comment by alexanderchan** [comment]
thanks @tjwoon for helping answer this.  Closing

**Comment by adochiny** [comment]
Thanks that also helped me.

**Comment by alain1987** [comment]
Thank you @tjwoon, your answer was really helpfull

**Comment by ravimule2277** [comment]
Thanks @tjwoon it works.
